src/bos/operators/utils/clients/ims.py

```diff
@@ -48,6 +55,38 @@ def __init__(self, image_id: str):
         super().__init__(f"IMS image id '{image_id}' does not exist in IMS")
 
 
+def get_image(image_id, session=None) -> dict:
```

Comment:
I encourage you to use type annotations for the input parameters, too.

Reply:
Yeah, I originally didn't because the functions that already existed in this file didn't use type hinting, but that's no excuse.